src/renderer/pages/Operations/components/LogModal.tsx

```diff
@@ -35,11 +36,20 @@ const EventMessage: Partial<Record<SigningStatus | 'INITIATED', string>> = {
   ERROR_CANCELLED: 'log.errorCancelledMessage',
 } as const;
 
+const getFilteredWalletsMap = (wallets: Wallet[]): Map<Wallet['id'], Wallet> =>
+  new Map(
```

Comment:
Filling object with a single `reduce` is more straightforward
Also use utility function for this `w.type === WalletType.MULTISHARD_PARITY_SIGNER`